src/Smartsheet.php

```diff
@@ -37,27 +37,54 @@ private function __construct( array $columns, array $rows ) {
 	 * @return static The instance.
 	 * @throws InvalidArgumentException If the data doesn't have the correct keys.
 	 */
-	public static function from_api_response( array $data ): self {
+	public static function from_smartsheet_api_response( array $data ): self {
 		if ( ! isset( $data['columns'], $data['rows'] ) ) {
 			throw new InvalidArgumentException( 'Cannot create from API data as it does not have rows.' );
 		}
 
 		return new self( $data['columns'], $data['rows'] );
 	}
 
+	/**
+	 * Create sheet from Google API response. For now this converts it to the format Smartsheet's API returned, so that
+	 * we need to convert less code. When Smartsheet is removed we can decouple it from this structured.
+	 *
+	 * @param array $header Row of values with the headers.
+	 * @param array $rows All the other rows.
+	 *
+	 * @return Smartsheet A sheet with the values.
+	 */
+	public static function from_google_response( array $header, array $rows ): Smartsheet {
+		$columns = array_map( fn( $h ) => [ 'title' => $h ], $header );
+
+		$convert_cell = fn( $c ) => [ 'value' => $c ];
+		$convert_row  = fn( $r ) => [ 'cells' => array_map( $convert_cell, $r ) ];
+
+		$rows = array_map( $convert_row, $rows );
+
+		return new self( $columns, $rows );
+	}
+
 	/**
 	 * Remove all rows that don't match the value in a specific column.
 	 *
 	 * @param int   $column_index Index of the column to use for filtering.
-	 * @param mixed $column_value The column value to use for filtering.
+	 * @param mixed $expected The column value to use for filtering.
+	 * @param bool  $convert_boolean Convert to boolean based on values being truthy.
 	 *
-	 * @return Smartsheet
+	 * @return Smartsheet The filtered sheet.
 	 */
-	public function filter_by_column( int $column_index, $column_value ): self {
+	public function filter_by_column( int $column_index, $expected, bool $convert_boolean = false ): self {
 		$rows = array_filter(
 			$this->rows,
-			function ( $row ) use ( $column_index, $column_value ) {
-				return $row['cells'][ $column_index ]['value'] === $column_value;
+			function ( $row ) use ( $convert_boolean, $column_index, $expected ) {
+				$value = $row['cells'][ $column_index ]['value'];
+
+				if ( $convert_boolean ) {
+					return ! $expected === ! $value;
```

Comment:
Apparently this [trips out the Yoda conditions check](https://app.circleci.com/pipelines/github/greenpeace/planet4-master-theme/4384/workflows/47ff3711-ce46-4539-9d17-477e08fdad39/jobs/25602), which fails no matter which one is first.